components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1603,6 +1603,25 @@ _resolve_dependencies() {
   _validate_deps
 }
 
+function do_automatic() {
+  do_default_automatic "$1"
+  return $?
+}
+
+# Loads specified automatic to extend plan DSL.
+function do_default_automatic() {
+  local automatic="${automatic:-$1}"
```

Comment:
Should this be `pkg_automatic`? I think that's what we had in the docs--makes things more consistent for the Plan author ("all plan variables are prefixed with `pkg_`"). Given that this variable comes from a `plan.sh`, we should be able to use this variable outright with a local or checking its value.